src/main/webapp/app/exercises/modeling/manage/modeling-exercise.service.ts

```diff
@@ -77,10 +77,11 @@ export class ModelingExerciseService implements ExerciseServicable<ModelingExerc
             .pipe(map((response: HttpResponse<ModelingPlagiarismResult>) => response.body!));
     }
 
-    convertToPdf(model: string, filename: string): Observable<any> {
+    convertToPdf(model: string, filename: string): Observable<HttpResponse<Blob>> {
         return this.http
             .post(`${SERVER_API_URL}api/apollon-convert/pdf`, { model }, { observe: 'response', responseType: 'blob' })
-            .pipe(map((response: HttpResponse<Blob>) => downloadStream(response.body, 'application/pdf', filename)));
+            .pipe(tap((response: HttpResponse<Blob>) => downloadStream(response.body, 'application/pdf', filename)))
+            .pipe(tap((foo) => foo));
```

Comment:
Why is this needed? Seems a bit out of place to me.

Reply:
Whoopsie. Looks like I pushed a debugging thingy